accepted/2021/logging-generator.md

```diff
@@ -333,6 +332,162 @@ same console logs formatted using JsonConsole:
 }
 ```
 
+### Added benefits with source generators not supported with `LoggerMessage.Define`
```

Comment:
I don't really understand this. This feels like a choice. We chose not to add those overloads. If we want to support more than 6 overloads then we need to add those overloads...

Reply:
even if there were overloads up to taking 16 parameter (Action limit). then the source generator would still be able to fallback to another approach to support arbitrary number of parameters

I think with this section I was trying to illustrate that source generator can deal with some of the existing limitations of `Define`, one way or another, if we wanted to allow more robust support. (arbitrary number of parameter is one of the several items of limitations explained in this section)

Reply:
Are recommending to switch that ability off for preview 4, to only allow up to 6 parameters until we find request for it?

Reply:
> Are recommending to switch that ability off for preview 4, to only allow up to 6 parameters until we find request for it?

I think so. I'm not sure why the API was rejected TBH. Maybe we're waiting until customers complain 😄